### ISSUE TITLE
brewmaster monk no aoe

### ISSUE THREAD

**Issue by Moonst** [BadBoy-Ultimate-Raider/BadBoy#125]
I see Chumii is working on the brewmaster mnk rotation and i gotta say its awsome just a few things i notice when i change from xuen for single target or basic cleave to rushing jade wind it doesnt notice the change also i do reload ui and there is no aoe really that is working for it not sure if it was ment this way or any thing but roation seems very solid i have seen some pauses in it though and when i tank an encounter such as Kromog in BRF when i get into the hands it wont dps them also touch of death does not go off. Chumii youor doing an awsome job and ill test the hell outta this if you want me to =P

**Comment by Moonst** [comment]
was also wondering if useing rushing jade wind can it be worked into the rotation like a normal ability regardless of how many adds are around and fire breat only goes if there are 2 or more adds? just a suggestion but really is very well written thank you again

**Comment by Moonst** [comment]
Hey chumii umm seems aoe is broken ive reloaded and such ill try a few more things i also did do a /reload and still did not fix it but would greatly appreciate the rushing jade winds ability to work on single target too the multi strike crits are nice to proc healing globs =P any how thnx again

**Comment by chumiiger** [comment]
sry, didnt look at the issues for quite a while

aoe works fine for me, it should trigger as soon as there are 3+ enemies in 8y range. I am counting enemies with some old function from times before enemies engine etc, so there is a better way to do it now.. i will look over general changes in simcraft files anyways, will look at aoe along with that over the weekend i think.
someelese reported the rotation to get stuck sometimes already and i noticed it too yesterday. the rotation will go on as soon as you move so i think i made some mistake with movement true/false on some abilites. will look into that too
regarding RJW in single target.. since multistrike is no stat you would go for, i dont really see the point in adding it to the single target rotation since you could use the energy otherwise. what i can do is a toggle so anyone could decide for themselves, but i would need to know what priority it should have then.

**Comment by Moonst** [comment]
on single target its low priority and thank you very much for checking it out and the toggle would be awsome thank you again very much for the great work

**Comment by chumiiger** [comment]
So, I updated the profile right now.
Added a toggle to use RJW on single target, priority will be above Jab/Tiger Palm cause otherwise you wont ever have the energy to cast it (due to my testing).
Also updated some things for 6.1 simc changes. Shuffle uptime is much better now.
Guard will now always safe a stack. (It will only use Guard at 2 stacks or if recharge time of the 2nd stack is < 5 seconds. I will test it this week, dont know if I like it like this but thats how simc is doing it, so maybe I will change it back again to use on cooldown, will see).
Regarding AoE.. I still cant reproduce your problem. AoE is working fine for me. It should trigger as soon as there are 3+ enemies in 8y range. Also I dont have problems when switching between RJW and Xuen.. pls test again, maybe I fixed it without knowing ;)
